data/prod/news/c0000000025o.json

```diff
@@ -125,7 +125,7 @@
                   "height": 576,
                   "locator": "/cpsprodpb/A933/production/_101651334_bouquet_pa.jpg",
                   "originCode": null,
-                  "copyrightHolder": "BBC"
+                  "copyrightHolder": "Getty Images"
```

Comment:
@bcmn The production articles are ones which we will *actually* put in production - they'll be audience-facing. For this reason, we shouldn't change the image attribution to be inaccurate.  

@jimjohnsonrollings Jim - is there an example of an image on the Prod articles that have non-BBC attribution, which we could use for testing?

Reply:
Yes - 25 has both BBC and non-BBC images in.

Reply:
Great, we'll add them, then.

Reply:
Updated prod pages to reflect real copyrightHolders & changed cypress tests to reflect.